cardano-cli/src/Cardano/CLI/Options/Flag.hs

```diff
@@ -0,0 +1,99 @@
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE OverloadedRecordDot #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE NoFieldSelectors #-}
+
+module Cardano.CLI.Options.Flag
```

Comment:
We're using singular nouns or module names
```suggestion
module Cardano.CLI.Option.Flag
```

Reply:
Fixed!